roles/ceph-common/tasks/installs/install_rh_storage_on_redhat.yml

```diff
@@ -0,0 +1,140 @@
+---
+- name: install red hat storage repository key
+  rpm_key:
+    key: "{{ ceph_stable_rh_storage_repository_path }}/RPM-GPG-KEY-redhat-release"
+    state: present
+  when:
+    - ceph_stable_rh_storage_iso_install
+
+- name: add red hat storage repository
+  template:
+    src: ../../templates/redhat_storage_repo.j2
+    dest: /etc/yum.repos.d/rh_storage.repo
+    owner: root
+    group: root
+    mode: 0644
+  when:
+    - ceph_stable_rh_storage_iso_install
+
+- name: install dependencies
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items: redhat_package_dependencies
+  when:
+    - ansible_distribution == "RedHat"
```

Comment:
But the dependencies are different from redhat to centos, we have both:

* `redhat_package_dependencies`
* `centos_package_dependencies`